components/dashboard/src/teams/sso/OIDCClientListItem.tsx

```diff
@@ -111,6 +166,33 @@ export const OIDCClientListItem: FC<Props> = ({ clientConfig }) => {
             {showEditModal && (
                 <OIDCClientConfigModal clientConfig={clientConfig} onClose={() => setShowEditModal(false)} />
             )}
+            {showActivateModal && (
+                <ConfirmationModal
+                    title="Activate SSO configuration"
+                    areYouSureText="Are you sure you want to activate the following SSO configuration?"
+                    children={{
+                        name: clientConfig.oidcConfig?.issuer ?? "",
+                        description: clientConfig.oauth2Config?.clientId ?? "",
+                    }}
+                    buttonText="Activate"
+                    buttonType="primary"
+                    buttonLoading={activateClient.isLoading}
+                    warningText={
+                        hasActiveConfig
+                            ? "Activating this SSO configuration will also deactivate the currently active configuration."
```

Comment:
**suggestion**: This is minor but I'd move this to the top of the modal as the warning affects the whole action not parts of the modal information. WDYT?

| BEFORE | AFTER |
|-|-|
| <img width="1512" alt="Screenshot 2023-05-16 at 23 12 22" src="https://github.com/gitpod-io/gitpod/assets/120486/16cb6b24-ebf1-4b06-b480-781feb9f7ebd"> | <img width="1512" alt="Screenshot 2023-05-16 at 23 12 33" src="https://github.com/gitpod-io/gitpod/assets/120486/a9501f4c-a25f-4275-8cac-e5610265fcc5"> |

Reply:
Makes sense to me I think - I was just using the existing `<ConfirmationModal/>` component and this is how it renders. I could look at flipping it in `ConfirmationModal` though so it's reflected everywhere we use it. Thoughts?